build_library/grub_install.sh

```diff
@@ -118,8 +123,22 @@ EOF
 
 if [[ ! -f "${ESP_DIR}/coreos/grub/grub.cfg.tar" ]]; then
     info "Generating grub.cfg memdisk"
+
+    grub_temp_dir=$(mktemp -d)
```

Comment:
easiest way would be to make it a global var above and add to the cleanup function